lib/http_perform.c

```diff
@@ -331,6 +352,15 @@ sf_bool STDCALL http_perform(CURL *curl,
 
         log_trace("Running curl call");
         res = curl_easy_perform(curl);
+
+        // forcely trigger renew timeout on the first attempt
+        if ((renew_injection) && (renew_timeout > 0) &&
+            elapsed_time && (*elapsed_time <= 0))
+        {
+            my_sleep_ms(renew_timeout * 1000);
+            res = CURLE_OPERATION_TIMEOUTED;
```

Comment:
nit: *TIMEDOUT

Reply:
Will fix. Turns out both of them are defined in curl.h.